pandas/tests/test_window.py

```diff
@@ -4154,3 +4188,20 @@ def test_rolling_cov_offset(self):
 
         expected2 = ss.rolling(3, min_periods=1).cov()
         tm.assert_series_equal(result, expected2)
+
+
```

Comment:
not averse to actually putting the tests in a separate file (may make review easier).

so I think a pre-cursor PR to move pandas/tests/test_window.py -> pandas/tests/window/test_window.py makes sense (then we can add on things in an easy namespace)